### PR TITLE
out_datadog: custom http headers

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
Added docs for custom Datadog output custom HTTP headers

### CODE REVIEW

File: pipeline/outputs/datadog.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Before you begin, you need a [Datadog account](https://app.datadoghq.com/signup)
...
 | dd_message_key  | By default, the plugin searches for the key 'log' and remap the value to the key 'message'. If the property is set, the plugin will search the property name key.                                                                                                                                                                                                                                                                          |                                  |
 | workers | The number of [workers](../../administration/multithreading.md#outputs) to perform flush operations for this output. | `0` |
+| header | Add additional arbitrary HTTP header key/value pair. Multiple headers can be set. |  |
```
Comment (lockewritesdocs) [inline pipeline/outputs/datadog.md:29]:
```suggestion
| header | Add additional arbitrary HTTP header key/value pair. Multiple headers can be set. | _none_ |
```

Comment (lockewritesdocs) [reply pipeline/outputs/datadog.md:29]: @AndrewChubatiuk -- if no default value is set, we add _none_ to the column so users know there isn't a default value.

**Review by lockewritesdocs — CHANGES_REQUESTED** [review]
Left a minor suggested change -- thanks @AndrewChubatiuk!

**Comment by AndrewChubatiuk** [comment]
hey @lockewritesdocs 
applied your suggestion

**Review by lockewritesdocs — APPROVED** [review]
LGTM from a technical writing perspective.

**Pull request merged**